backend/mem/location_test.go

```diff
@@ -58,16 +57,16 @@ func (s *memLocationTest) TestList_NonExistentDirectory() {
 
 	contents, err := location.List()
 	s.Nil(err, "error isn't expected")
-	s.Equal(0, len(contents), "List should return empty slice for non-existent directory")
+	s.Equal(0, len(contents), "list should return empty slice for non-existent directory")
 
 	prefixContents, err := location.ListByPrefix("anything")
 	s.Nil(err, "error isn't expected")
-	s.Equal(0, len(prefixContents), "ListByPrefix should return empty slice for non-existent directory")
+	s.Equal(0, len(prefixContents), "listByPrefix should return empty slice for non-existent directory")
 
 	regex, _ := regexp.Compile("[-]+")
 	regexContents, err := location.ListByRegex(regex)
 	s.Nil(err, "error isn't expected")
-	s.Equal(0, len(regexContents), "ListByRegex should return empty slice for non-existent directory")
+	s.Equal(0, len(regexContents), "listByRegex should return empty slice for non-existent directory")
```

Comment:
List and ListByPrefix are function names and shouldn't be lowercased, even when starting an error message.